src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,38 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $watch-size;
```

Comment:
it is a good ability to create mixin or function  for circles  (width, height, radius)